agent/xds/serverless_plugin_oss_test.go

```diff
@@ -142,6 +159,15 @@ func TestServerlessPluginFromSnapshot(t *testing.T) {
 								}
 							},
 						},
+						{
+							name: "endpoints",
+							key:  xdscommon.EndpointType,
+							sorter: func(msgs []proto.Message) func(int, int) bool {
+								return func(i, j int) bool {
+									return msgs[i].(*envoy_endpoint_v3.ClusterLoadAssignment).ClusterName < msgs[j].(*envoy_endpoint_v3.ClusterLoadAssignment).ClusterName
+								}
+							},
+						},
```

Comment:
Currently, serverless plugin patchers can't update endpoints https://github.com/hashicorp/consul/blob/main/agent/xds/serverlessplugin/patcher.go#L15-L31. Adding tests for them doesn't hurt anything though.

Reply:
Yup, Dan had brought up adding them in the ENT PR for consistency across golden tests. I figured it would be good so we can ensure we don't change endpoints if that's the current expected behavior